privacyidea/lib/config.py

```diff
@@ -451,6 +452,9 @@ def get_token_class(tokentype):
             tokenclass = tclass
             break
 
+    if tokenclass is None:
+        raise ValidateError("invalid tokentype provided. ttype: {}".format(tokentype.lower()))
```

Comment:
If I would call 

    /ttype/ööäääh

raising the ValidateError would raise an EncodingError. So you should do

~~~~python
raise ParameterError(u"Invalid tokentype provided. ttype: {}".format(tokentype.lower()))
~~~~

Reply:
Adjusted